test/browser/browser.ini

```diff
@@ -5,8 +5,6 @@ support-files =
   head.js
 prefs =
   browser.newtabpage.activity-stream.debug=false
-# Temporarily disable the search improvements for mochitests until bug 1481680
-  browser.newtabpage.activity-stream.improvesearch.topSiteSearchShortcuts=false
```

Comment:
i think we should keep this, and then just enable it for the one test which we want to test the search shortcuts for... that way we don't have to keep disabling in the future when we add more mochitests that doesn't want this off, and the in the future when we enable this permanently we won't need to go in and remove all the disable code for each tests

Reply:
On the flip side, the default behavior is to have top site search shortcuts, so our tests should be closer to the default behavior

Reply:
Although looking at how the tests were fixed, they're disabling anyway… I wonder if the tests should work with it enabled instead

Reply:
Although then again... I would assume this pref is going away at some point.. soon? So we should not rely on being able to change the pref to false. ??

Reply:
I agree with that last part, I'm going to keep the pref on and update the test to expect the search topsites feature to exist.